docs/_docs/integrations/file-formats.md

```diff
@@ -30,6 +30,10 @@ The **VIEW_VULNERABILITY** permission is required to use the findings API.
 > Previous versions of Dependency-Track supported only a single CWE (`cweId` and `cweName` fields respectively) per
 > vulnerability.
 > The `cweId` and `cweName` fields are deprecated and will be removed in a later version. Please use `cwes` instead.
+ 
+> Finding Packaging Format v1.2 was introduced in Dependency-Track v4.7.2.
```

Comment:
Minor nitpick: It's currently unlikely that we'll release another bugfix version before 4.8.0.

```suggestion
> Finding Packaging Format v1.2 was introduced in Dependency-Track v4.8.0.
```

Reply:
That was just a guess from me :) fixed it: https://github.com/DependencyTrack/dependency-track/pull/2471/commits/dd4680fca6124b4eed9fb5da2d078128e3f156bb